proteuslib/tests/test_enrtl_water_pH.py

```diff
@@ -639,7 +636,7 @@ def test_solution_water(self, water_model):
                        #        we divide here by 1000**2 to perform both necessary
                        #        unit conversions in line.
                        "dh_rxn_ref": (0, pyunits.kJ/pyunits.mol),
-                       "k_eq_ref": (10**-14/1000/1000/55.2/55.2,pyunits.mol**2/pyunits.L**2),
+                       "k_eq_ref": (10**-14/1000/1000/55.2/55.2,pyunits.dimensionless),
                        "T_eq_ref": (298, pyunits.K),

```

Comment:
Remove all factors of 1000 in all k_eq_ref

Reply:
Whoops, that one got away, thanks for catching it.